tests are failing:

```
FAILED tests/www/test_views.py::TestAirflowBaseViews::test_code - jinja2.exce...
FAILED tests/www/test_views.py::TestAirflowBaseViews::test_code_from_db - jin...
FAILED tests/www/test_views.py::TestAirflowBaseViews::test_code_from_db_all_example_dags
FAILED tests/www/test_views.py::TestAirflowBaseViews::test_code_no_file - jin...
FAILED tests/www/test_views.py::TestAirflowBaseViews::test_dag_details - jinj...
FAILED tests/www/test_views.py::TestAirflowBaseViews::test_dag_details_subdag
FAILED tests/www/test_views.py::TestAirflowBaseViews::test_dag_details_trigger_origin_graph_view
FAILED tests/www/test_views.py::TestAirflowBaseViews::test_dag_details_trigger_origin_tree_view
FAILED tests/www/test_views.py::TestAirflowBaseViews::test_duration - jinja2....
FAILED tests/www/test_views.py::TestAirflowBaseViews::test_gantt - jinja2.exc...
FAILED tests/www/test_views.py::TestAirflowBaseViews::test_landing_times - ji...
FAILED tests/www/test_views.py::TestAirflowBaseViews::test_rendered_k8s - jin...
FAILED tests/www/test_views.py::TestAirflowBaseViews::test_rendered_template
FAILED tests/www/test_views.py::TestAirflowBaseViews::test_task - jinja2.exce...
FAILED tests/www/test_views.py::TestAirflowBaseViews::test_tries - jinja2.exc...
FAILED tests/www/test_views.py::TestAirflowBaseViews::test_view_uses_existing_dagbag_2_dag_details
FAILED tests/www/test_views.py::TestAirflowBaseViews::test_xcom - jinja2.exce...
FAILED tests/www/test_views.py::TestGanttView::test_dt_nr_dr_form_default_parameters
FAILED tests/www/test_views.py::TestGanttView::test_dt_nr_dr_form_with_base_date_and_num_runs_and_execution_date_outside
FAILED tests/www/test_views.py::TestGanttView::test_dt_nr_dr_form_with_base_date_and_num_runs_and_execution_date_within
FAILED tests/www/test_views.py::TestGanttView::test_dt_nr_dr_form_with_base_date_and_num_runs_parameters_only
FAILED tests/www/test_views.py::TestGanttView::test_dt_nr_dr_form_with_execution_date_parameter_only
FAILED tests/www/test_views.py::TestDagACLView::test_code_success - jinja2.ex...
FAILED tests/www/test_views.py::TestDagACLView::test_code_success_for_all_dag_user
FAILED tests/www/test_views.py::TestDagACLView::test_dag_details_success - ji...
FAILED tests/www/test_views.py::TestDagACLView::test_dag_details_success_for_all_dag_user
FAILED tests/www/test_views.py::TestDagACLView::test_duration_success - jinja...
FAILED tests/www/test_views.py::TestDagACLView::test_gantt_success - jinja2.e...
FAILED tests/www/test_views.py::TestDagACLView::test_landing_times_success - ...
FAILED tests/www/test_views.py::TestDagACLView::test_log_success - jinja2.exc...
FAILED tests/www/test_views.py::TestDagACLView::test_log_success_for_user - j...
FAILED tests/www/test_views.py::TestDagACLView::test_rendered_template_success
FAILED tests/www/test_views.py::TestDagACLView::test_rendered_template_success_for_all_dag_user
FAILED tests/www/test_views.py::TestDagACLView::test_task_success - jinja2.ex...
FAILED tests/www/test_views.py::TestDagACLView::test_task_success_for_all_dag_user
FAILED tests/www/test_views.py::TestDagACLView::test_tries_success - jinja2.e...
FAILED tests/www/test_views.py::TestDagACLView::test_xcom_success - jinja2.ex...
FAILED tests/www/test_views.py::TestDagACLView::test_xcom_success_for_all_dag_user
FAILED tests/www/test_views.py::TestRenderedView::test_rendered_template_view
FAILED tests/www/test_views.py::TestRenderedView::test_rendered_template_view_for_unexecuted_tis
FAILED tests/www/test_views.py::TestRenderedView::test_user_defined_filter_and_macros_raise_error
FAILED tests/www/test_views.py::TestExtraLinks::test_extra_link_in_gantt_view
```

--- a/airflow/www/templates/appbuilder/dag_docs.html
+++ b/airflow/www/templates/appbuilder/dag_docs.html
@@ -18,7 +18,7 @@
 #}
 
 {% macro dag_docs(doc_md) %}
-  {% if doc_md %}
+  {% if doc_md is defined %}
     <div class="panel-group" id="accordion" role="tablist" aria-multiselectable="true" style="margin-top: 16px;">
       <div class="panel panel-default">
         <div class="panel-heading" role="tab" id="headingOne">

